apache/trafficcontrol
# TP: adds the ability to delete a job (aka invalidation request)

- [x] This PR is related to #2030

Adds the ability in TP to delete an invalidate content request job. User is restricted to only deleting jobs within their tenancy, otherwise, only jobs associated with delivery services within their tenancy.

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?

- Navigate to https://traffic-portal.tld/#!/jobs. Delete a job by right-clicking on a job row. Note the delivery service of the job. Note the cdn of the delivery service.
- Navigate to https://traffic-portal.tld/#!/cdns/{cdn-ID}/servers. Note that the "Reval Pending" flag for all servers (except for OFFLINE or PRE_PROD) have been set to true showing a clock icon.

## The following criteria are ALL met by this PR
